WcaOnRails/app/views/polls/vote.html.erb

```diff
@@ -0,0 +1,31 @@
+<% provide(:title, 'Vote in poll #' + @poll.id.to_s) %>
```

Comment:
Do we actually want to show the poll id, or is this just development code? Is there something human readable we could say instead?